artemis/config.py

```diff
@@ -698,6 +698,28 @@ class Nuclei:
                 "NUCLEI_TEMPLATE_CHUNK_SIZE is 200, three calls will be made with 200 templates each.",
             ] = get_config("NUCLEI_TEMPLATE_CHUNK_SIZE", default=200, cast=int)
 
+        class PlaceholderPageContent:
+            PLACEHOLDER_PAGE_DETECTOR_ENABLE: Annotated[
+                bool,
+                "Enable or disable placeholder pages detector."
```

Comment:
add maybe "using this feature you may skip vulnerability scanning for websites that aren't built yet, but e.g. contain a hosting provider placeholder page"